material/tutorials.md

```diff
@@ -0,0 +1,11 @@
+# Tutorials
+
+## Python
```

Comment:
Perhaps also consider something like https://realpython.com/ (they have a ton of free resources, but (for the paid user) also really great (video) courses.

Reply:
Added it and I also love their content 😄